### PR TITLE
Allows the biogenerator in Hydroponics to print paper and fake space cash

### PR DESCRIPTION
**Pull request by ma44**
**FAKE CASH IS NOT WORTH ANYTHING**

:cl: ma44
add: Biogenerator can now make paper by using biomatter. 
add: A researcher has discovered you can make space cash bills that look exactly like the original ones, these can be made in the Biogenerator in hydroponics with biomatter. 
add: In response to this, the Centcomm in your sector has upgraded the ATM to prevent the fake space cash being used as money, exported money will also be checked to make sure they are not fake. 
/:cl:

This change is good

1. You can print dosh which doesn't effect game balance
2. Antags can use this to bribe or trade for items if the crew can roleplay
3. You can make a fun economy
4. Makes ma44 satisfy his space cash fetish

### CODE REVIEW

**Comment by ExcessiveUseOfCobblestone** [comment]
what's the point of it then

**Comment by Mindustry** [comment]
Should require the biogenerator to be hacked to print the fake money. Doesn't seem like something that would come standard with a machine.

Number 2 isn't really a good example because traitors can already purchase briefcases full of REAL money

**Comment by ma44** [comment]
Trick people into thinking it's real money, also making your own crappy economy

**Comment by Armhulen** [comment]
what's the point of it then

**Comment by jughu** [comment]
A economy with fake money is basically a economy with real money

**Comment by ma44** [comment]
So you can spend the thousands of biomatter on something that makes you feel rich

**Comment by coiax** [comment]
Modify the Service borg cash fabricator to make fake cash then.

**Comment by jughu** [comment]
No keep service borg cash real so i can murderbone as qm and make a cash printery and ship it

**Comment by jughu** [comment]
Or make it real when emagged

**Comment by Mindustry** [comment]
also merging from master ayy

**Comment by ma44** [comment]
The service borg produces 20 credit bills, nothing to worry about

**Comment by jughu** [comment]
Thats 20 points per produced dosh

**Comment by jughu** [comment]
Yay now i can make more paper which get rarely used anyways!

**Comment by Cruix** [comment]
But what about the biogenerator in perma?

**Comment by ma44** [comment]
What would you do with fake cash and paper?

**Comment by Aurantiaco** [comment]
Bribe the security officer to be let out.

**Comment by Mindustry** [comment]
They'll just meta-know the cash is fake.

**Comment by ma44** [comment]
If it was real, that possible cash could be used for some fun WT550s or tasers

**Comment by ExcessiveUseOfCobblestone** [comment]
you don't just have normal stacks of cash in perma...

File: code/game/objects/items/stacks/cash.dm [hunk 1]
```diff
@@ -46,4 +46,53 @@
...
+	throwforce = 0
+	throw_speed = 2
+	throw_range = 2
+	w_class = WEIGHT_CLASS_TINY
+	resistance_flags = FLAMMABLE
+	
+/obj/item/stack/spacecash/fake/c10
+	icon_state = "spacecash10"
+	desc = "It's worth 10 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: Duplicate Definition.

File: code/game/objects/items/stacks/cash.dm [hunk 2]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c20
+	icon_state = "spacecash20"
+	desc = "It's worth 20 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: And here.

File: code/game/objects/items/stacks/cash.dm [hunk 3]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c50
+	icon_state = "spacecash50"
+	desc = "It's worth 50 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: And here.

File: code/game/objects/items/stacks/cash.dm [hunk 4]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c100
+	icon_state = "spacecash100"
+	desc = "It's worth 100 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: And here.

File: code/game/objects/items/stacks/cash.dm [hunk 5]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c200
+	icon_state = "spacecash200"
+	desc = "It's worth 200 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: And here.

File: code/game/objects/items/stacks/cash.dm [hunk 6]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c500
+	icon_state = "spacecash500"
+	desc = "It's worth 500 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: And here.

File: code/game/objects/items/stacks/cash.dm [hunk 7]
```diff
@@ -46,4 +46,53 @@
...
+
+/obj/item/stack/spacecash/fake/c1000
+	icon_state = "spacecash1000"
+	desc = "It's worth 1000 credits."
+	value = 0
```
Comment (francinum) [inline code/game/objects/items/stacks/cash.dm]: You get the point.

**Review by francinum — CHANGES_REQUESTED** [review]

**Comment by jughu** [comment]
When is paper gonna be able to be stacked

**Comment by kevinz000** [comment]
ur not worth anything lolol












 jk ily man.

**Pull request closed without merging**